Makefile

```diff
@@ -1,13 +1,17 @@
-.PHONY: attach build build_tester clean coverage create_network psql release_pypi release_pypitest release_quay remove_network start_postgres stop_postgres test test27 test36 wait_for_postgres
+.PHONY: attach build build_tester clean coverage create_network psql release_pypi release_pypitest release_quay remove_network start_postgres stop_postgres test test_one_pg_version test27 test36 wait_for_postgres
+
+SUPPORTED_PG_VERSIONS= 9.5.13 9.6.4
```

Comment:
This might be better off as a `?=` so that the travis build can parametrize it and runs separate builds per postgres version.